### DISCUSSION TITLE
Way to increase delay on ffmpeg conversion

### DISCUSSION THREAD

**Discussion by efraimjer** [bluenviron/mediamtx#3577] [Questions]
### Question

Hello I'm using mediamtx to encode some security cameras and add a watermark and crop the video with ffmpeg, but I need to increase the already existing conversion delay in 10 seconds to sync with a button the will be pressed, is there a way to do this?

**Comment by aler9** [comment]
Hello, instructions on how to add intentional, additional delay when re-encoding streams are here: https://github.com/bluenviron/mediamtx/issues/763#issuecomment-1883023838
